cat-test/tests/cat_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import 'package:cat_test/cat_cafe.dart';
+import 'package:cat_test/cat.dart';
+
+@proxy
+class MockCatCafe extends Mock implements CatCafe {}
+
+class MockCat extends Mock implements Cat {}
+
+main() {
+  test('cat cafe', () {
```

Comment:
I usually try to make test descriptions full sentences, like "Cat.getFood() returns food from the cat cafe"